applets/clock/clock-face.c

```diff
@@ -407,11 +403,11 @@ remove_pixbuf_from_cache (const char *key,
 static void
 clock_face_load_face (ClockFace *this, gint width, gint height)
 {
-        ClockFacePrivate *priv = CLOCK_FACE_GET_PRIVATE (this);
-	const gchar *size_string[2] = { "small", "large" };
-        const gchar *daytime_string[4] = { "morning", "day", "evening", "night" };
-	gchar *cache_name;
-	gchar *name;
+    ClockFacePrivate *priv = clock_face_get_instance_private (this);
+    const gchar *size_string[2] = { "small", "large" };
+    const gchar *daytime_string[4] = { "morning", "day", "evening", "night" };
+    gchar *cache_name;
+    gchar *name;
```

Comment:
wrong indentation

Reply:
Fixed and force-pushed to keep things moving

Reply:
no, it isn't fixed

see now in master https://github.com/mate-desktop/mate-panel/blob/master/applets/clock/clock-face.c#L410

Reply:
fixed by https://github.com/mate-desktop/mate-panel/pull/1018